src/go/rpk/pkg/cli/cmd/container/start.go

```diff
@@ -103,25 +117,28 @@ func startCluster(
 		return nil
 	}
 
-	log.Info("Downloading latest version of Redpanda")
-	err = common.PullImage(c)
-	if err != nil {
-		log.Debugf("Error trying to pull latest image: %v", err)
-
-		msg := "Couldn't pull image and a local one wasn't found either."
-		if c.IsErrConnectionFailed(err) {
-			msg += "\nPlease check your internet connection" +
-				" and try again."
-		}
-
-		present, checkErr := common.CheckIfImgPresent(c)
-
-		if checkErr != nil {
-			log.Debugf("Error trying to list local images: %v", err)
-
-		}
-		if !present {
-			return errors.New(msg)
+	log.Debug("Checking for a local image.")
```

Comment:
Is intentional that the cluster must be purged, otherwise `--image` is ignored?

Reply:
Yep. `rpk container` only supports a single cluster currently.